bin/_util.js

```diff
@@ -472,5 +506,54 @@ module.exports = {
     get value() {
       return this.parseResolvedValue(this.getResovledValue());
     }
+
+    get valueRef() {
+      switch (this._figmaVariableData.resolvedType) {
+        case 'COLOR':
+          // TODO: (in source) transparent exists as a tier 1 token but should not
+          if (this.getResovledName() === 'Render/Transparent') {
+            return 'transparent';
+          }
+          return module.exports.FigmaVariable.isAliased(
+            this._figmaVariableData,
+            this._mode,
+          )
+            ? `{${this._tokenNameToPath(
+                this.getResovledName()
+                  .replace(
+                    module.exports.FigmaVariable.TIER_1_PREFIX,
+                    'eds.color.',
+                  )
+                  // TODO: (in source) remove duplicate color from structures
+                  .replace('Neutral/Neutral', 'Neutral')
+                  .replace('Red/Red', 'Red')
+                  .replace('Orange/Orange', 'Orange')
+                  .replace('Yellow/Yellow', 'Yellow')
+                  .replace('Green/Green', 'Green')
+                  .replace('Blue/Blue', 'Blue')
+                  .replace('Purple/Purple', 'Purple')
+                  .replace('Pink/Pink', 'Pink')
+                  // TODO: (in source) lower case the names of the tier 1 color tokens
+                  .toLowerCase(),
```

Comment:
TODOs with (in source) are changes to make in Figma for consistency. This will get removed once those are done.